function-server/src/test/java/io/dispatchframework/javabaseimage/ServerTest.java

```diff
@@ -0,0 +1,81 @@
+package io.dispatchframework.javabaseimage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Type;
+import java.util.function.BiFunction;
+
+public class ServerTest {
```

Comment:
General comment on all the test methods. Declare on each of them `throws Exception` if any exceptions need to be thrown within the test body (no need to list all specific ones). It makes easier to write and read the tests. If any exception actually gets thrown, the test will fail anyway.